unit/suites/test_execute.py

```diff
@@ -0,0 +1,94 @@
+# -*- coding: utf-8 -*-
+
+from __future__ import print_function
+
+import sys
+import unittest
+
+import tarantool
+from .lib.tarantool_server import TarantoolServer
+
+
+class TestSuite_Execute(unittest.TestCase):
+    table_prefix = 'dbapi20test_'  # If you need to specify a prefix for tables
+
+    ddl = 'create table %s (id INTEGER PRIMARY KEY AUTOINCREMENT, ' \
+           'name varchar(20))'
+
+    ddl_params = [
+        {'id': None, 'name': 'Michael'},
+        {'id': None, 'name': 'Mary'},
+        {'id': None, 'name': 'John'},
+        {'id': None, 'name': 'Ruth'},
+        {'id': None, 'name': 'Rachel'}
+    ]
+
+    @classmethod
+    def setUpClass(self):
+        print(' EXECUTE '.center(70, '='), file=sys.stderr)
+        print('-' * 70, file=sys.stderr)
+        self.srv = TarantoolServer()
+        self.srv.script = 'unit/suites/box.lua'
+        self.srv.start()
+        self.con = tarantool.Connection(self.srv.host, self.srv.args['primary'])
+
+    def setUp(self):
+        # prevent a remote tarantool from clean our session
+        if self.srv.is_started():
+            self.srv.touch_lock()
+        self.con.flush_schema()
+
+        # grant full access to guest
+        self.srv.admin("box.schema.user.grant('guest', 'create,read,write,"
+                       "execute', 'universe')")
+
+    @classmethod
+    def tearDownClass(self):
+        self.con.close()
+        self.srv.stop()
+        self.srv.clean()
+
+    def _populate_data(self, table_name):
+        query = "insert into %s values (:id, :name)" % table_name
+        for param in self.ddl_params:
+            self.con.execute(query, param)
+
+    def _create_table(self, table_name):
+        return self.con.execute(self.ddl % table_name)
+
+    def _create_table_name(self, name):
+        return self.table_prefix + name
+
+    def test_dml_response(self):
+        table_name = self._create_table_name('foo')
+        response = self._create_table(table_name)
+        self.assertEqual(response.rowcount, 1)
+        self.assertEqual(response.lastrowid, None)
+        self.assertEqual(response.data, None)
+
+        query = "insert into %s values (:id, :name)" % table_name
+
+        for num, param in enumerate(self.ddl_params, start=1):
+            response = self.con.execute(query, param)
+            self.assertEqual(response.lastrowid, num)
+            self.assertEqual(response.rowcount, 1)
+            self.assertEqual(response.data, None)
```

Comment:
Aren't you want to verify `response.rows`? The API has three properties and here we test two of them.

Reply:
I removed .rows property because it has the same behaviour as .data property